src/POMDPToolbox.jl

```diff
@@ -11,11 +11,13 @@ import POMDPs: action, value, solve
 import POMDPs: actions, action_index, state_index, obs_index, iterator, sampletype, states, n_actions, n_states, observations, n_observations, discount, isterminal
 import POMDPs: generate_sr, initial_state
 import Base: rand, rand!, mean, ==
+import DataStructures: CircularBuffer, isfull, capacity, push!, append!
```

Comment:
We don't need this since we're not using circular buffers anymore, so make sure to take it out